File: un6unw/Week32/숫자 게임.java
import java.util.*;

class Solution {
    public int solution(int[] A, int[] B) {
        Arrays.sort(A); // A 오름차순 정렬
        Arrays.sort(B); // B 오름차순 정렬

        int aIndex = 0;
        int bIndex = 0;
        int score = 0;

        // B의 숫자가 A의 숫자보다 클 때만 점수 획득
        while (aIndex < A.length && bIndex < B.length) {
            if (B[bIndex] > A[aIndex]) {
                score++;
                aIndex++;
                bIndex++;
            } else {
                bIndex++;
            }
        }

        return score;
    }
}

File: oyatplum/Week32/숫자 게임.py
def solution(A, B):
    answer = 0
    A.sort()
    B.sort()
    a = 0
    b = 0

    while a < len(A) and b < len(B):
        if A[a] >= B[b]:
            b += 1
            continue
        else:
            answer += 1
            a += 1
            b += 1

    return answer